distribution/std-lib/Standard/src/Table/Data/Column.enso

```diff
@@ -228,12 +228,20 @@ type Column
         Nothing -> Error.throw Table.No_Index_Set_Error
         i -> Column i
 
-    ## Returns the item contained in this column at the given index.
-    at : Integer -> Any
+    ## Returns the value contained in this column at the given index.
+
+       If the value is an NA then this method returns nothing. If the index is
+       not an index in the column it returns an `Index_Out_Of_Bounds_Error`.
+
+       Arguments:
+       - `index`: The index in the column from which to get the value
+    at : Integer -> (Any | Nothing) ! Index_Out_Of_Bounds_Error
     at index =
-        storage = this.java_column.getStorage
-        if storage.isNa index then Nothing else
-            storage.getItem index
+        valid_index = (index >= 0) && (index < this.length)
+        if valid_index.not then Error.Throw (Index_Out_Of_Bounds_Error index this.length) else
```

Comment:
Shouldn't this be `Error.throw` and not `Throw`?

I see that there is a test for that, how did that pass CI? :open_mouth:

Reply:
Magic.